### ISSUE TITLE
PyCondorPlugin checking condor log too often

### ISSUE THREAD

**Issue by amaltaro** [dmwm/WMCore#6690]
While investigating my last fix to pycondorplugin, I noticed it fails to get job info from the schedd for one or 2 jobs out of 1k, and then it fallback to the condor log, e.g.:

```
2016-03-11 10:17:51,398:139714205484800:DEBUG:PyCondorPlugin:Opening condor job log file: /data/srv/wmagent/v1.0.14.pre8/install/wmagent/JobCreator/JobCache/amaltaro_SUS-RunIIWinter15pLHE-00232_00153_v2__160309_151840_9882/Production/JobCollection_2323_0/job_17580/condor.8561.11.log
2016-03-11 10:17:51,414:139714205484800:DEBUG:PyCondorPlugin:/data/srv/wmagent/v1.0.14.pre8/install/wmagent/JobCreator/JobCache/amaltaro_SUS-RunIIWinter15pLHE-00232_00153_v2__160309_151840_9882/Production/JobCollection_2323_0/job_17580/condor.8561.11.log is CORRUPT
2016-03-11 10:17:51,415:139714205484800:INFO:PyCondorPlugin:Retrieved 0 Info from Condor Job Log file /data/srv/wmagent/v1.0.14.pre8/install/wmagent/JobCreator/JobCache/amaltaro_SUS-RunIIWinter15pLHE-00232_00153_v2__160309_151840_9882/Production/JobCollection_2323_0/job_17580/condor.8561.11.log
2016-03-11 10:17:51,415:139714205484800:DEBUG:PyCondorPlugin:No job log Info for jobid=17580. Assume it is Complete. Check DB.
```

I have to check whether it comes from my patch, and if not we need to keep an eye on the impact in large scale.
From what I understood, when a job gets completed, the schedd fails to provide info for that job - as expected - and then the component tries reading the log file.

**Comment by hufnagel** [comment]
Isn't going to the condor log a fallback ? Wasn't there a way to get information about a recently completed job from memory ? Not sure how long this is buffered though...

**Comment by amaltaro** [comment]
Well, since you read this thread, do you from time to time see this message in JobStatusLite "Retrieved 0 Info from Condor Job..."?

Yes, reading the condor logs is a fallback mechanism. I always thought this was the schedd log file, but from a quick look at the code, it seems to be the condor.JOBID.log file instead.

**Comment by amaltaro** [comment]
Fixed by #7210
